dockerfiles/Jenkinsfile

```diff
@@ -0,0 +1,26 @@
+@Library("jenkins-library@main")
+
+import com.logicalclocks.jenkins.k8s.ImageBuilder
+
+
+node("local") {
+    stage('Clone repository') {
+      checkout scm
+    }
+
+    stage('Build and push image(s)') {
+      withCredentials([usernamePassword(credentialsId: 'a0770738-4ef3-4acc-a6ba-097ee6c85b44', passwordVariable: 'PASSWORD', usernameVariable: 'USERNAME')]) {
+        version = readFile "${env.WORKSPACE}/dockerfiles/HIVE_VERSION"
```

Comment:
we should get the version from the pom file similar to what we do here 
https://github.com/logicalclocks/clusterj-onlinefs/blob/master/Jenkinsfile#L18